gaphor/UML/classes/dependencypropertypages.py

```diff
@@ -10,8 +10,50 @@
 from gaphor.UML.classes.dependency import DependencyItem
 
 
-@PropertyPages.register(DependencyItem)
+@PropertyPages.register(Dependency)
```

Comment:
Hi @pbrown12303, could you explain more why having a Property Page unique for the Dependency is needed? I think for the other relationships and elements we only have a Property Page for the item.

Reply:
The split is only necessary when the you  want to display the properties that are owned by the diagram element (the item) in addition to those of he subject model element. The need for the split into two property pages becomes apparent when a subject model element is selected in the tree. If you try and use a property page that shows the diagram element property, you'll get an error because the subject model element does not have such a property. 

My recommendation would be to think of the intent of the current property page to be a display of just the subject model element's properties and name it accordingly (which I think is by and large the case right now). Only when you want to additionally display properties belonging specifically to the selected diagram element item would you need a second property page specific to the diagram element's item properties. 

In this particular case, the property page (before my change) was showing only diagram element item properties. No properties of the subject diagram element were shown. In keeping with the thinking I just described, I re-named the existing property page to indicate that it is item specific and created a new property page to display the subject element properties.

Reply:
@amolenaar, since you are much more the expert on how the PropertyPages should work, what do you think, do you we need a separate PropertyPage to be able to show the Source and Target?

Reply:
Done